### PR TITLE
schema: minor docs changes

### PR DESCRIPTION
**Pull request by oliver-sanders**
Currently poking around the schema looking at possible ways out of the graphql deadlock. Noticed some small things:

* Consistent string quotation.
* Consistent termination punctuation.
* Correct some ReST literals (double backquotes).
* Correct one incorrectly closed markdown literal.

**Check List**

- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Applied any dependency changes to both `setup.cfg` (and `conda-environment.yml` if present).
- [x] Tests are included (or explain why tests are not needed).
- [x] Changelog entry included if this is a change that can affect users
- [x] [Cylc-Doc](https://github.com/cylc/cylc-doc) pull request opened if required at cylc/cylc-doc/pull/XXXX.
- [x] If this is a bug fix, PR should be raised against the relevant `?.?.x` branch.

### CODE REVIEW

File: cylc/flow/network/schema.py [hunk 1]
```diff
@@ -985,35 +987,36 @@ def resolve_task_id(root, info, **args):
 
 class Prerequisite(ObjectType):
     class Meta:
-        description = """Task prerequisite."""
+        description = 'Task prerequisite.'
     expression = String()
     conditions = graphene.List(
         Condition,
-        description="""Condition monomers of a task prerequisites.""")
+        description='Condition monomers of a task prerequisites.'
```
Comment (hjoliver) [inline cylc/flow/network/schema.py:994]:
```suggestion
        description='Condition monomers of a task prerequisite.'
```

File: cylc/flow/network/schema.py [hunk 2]
```diff
@@ -2224,7 +2229,7 @@ def delta_subs(root, info: 'ResolveInfo', **args) -> AsyncGenerator[Any, None]:
 
 class Pruned(ObjectType):
     class Meta:
-        description = """WFS Nodes/Edges that have been removed."""
+        description = 'WFS Nodes/Edges that have been removed.'
```
Comment (hjoliver) [inline cylc/flow/network/schema.py:2232]: What's "WFS" ?  Workflow Service?   Should be "scheduler"? 

```
> git grep WFS
cylc/flow/network/schema.py:        description = """WFS Nodes/Edges that have been removed."""
cylc/flow/network/schema.py:        description = """Grouped deltas of the WFS publish"""
```

**Review by hjoliver — APPROVED** [review]

File: cylc/flow/network/schema.py (hunk 2 above)
Comment (oliver-sanders) [reply cylc/flow/network/schema.py:2232]: Absolutely no idea!

Comment (oliver-sanders) [reply cylc/flow/network/schema.py:2232]:
```suggestion
        description = sstrip('''
            Objects (e.g. workflows, tasks, jobs) which have been removed from
            the store.       
        ''')   
```

File: cylc/flow/network/schema.py (hunk 1 above)
Comment (oliver-sanders) [reply cylc/flow/network/schema.py:994]: "Condition monomers" - that's a good one!

File: cylc/flow/network/schema.py (hunk 2 above)
Comment (hjoliver) [reply cylc/flow/network/schema.py:2232]: Ping @dwsutherland - I think you may have been the original author?

Comment (oliver-sanders) [reply cylc/flow/network/schema.py:2232]: This is the description for pruned deltas.

These list Cylc "objects" that have drifted out of the window.

I expect WFS probably meant Workflows?

Comment (hjoliver) [reply cylc/flow/network/schema.py:2232]: I have feeling it date backs to early Cylc 8 planning days when we were considering various other names for what is now just the "scheduler" (Workflow Service = WFS?).

Comment (dwsutherland) [reply cylc/flow/network/schema.py:2232]: Yeah.. Can't recall 😆 

But "Workflow's" might have been it ...

Comment (hjoliver) [reply cylc/flow/network/schema.py:2232]: Ok screw it, merge Oliver's tweak.

File: cylc/flow/network/schema.py (hunk 1 above)
Comment (hjoliver) [reply cylc/flow/network/schema.py:994]: Yeah who's the wordsmith responsible for that gem?

**Review by dwsutherland — APPROVED** [review]
LGTM

**Comment by dwsutherland** [comment]
> ways out of the graphql deadlock.

I think we need to move to more active projects, i.e. only graphql-core, graphene/graphene-tornado  and drop graphql-ws by creating our own solution (that can be used in zmq too).. Or see if there's any other viable graphql ecosystem out there..

The previous generation of graphql-core/graphql-ws is stuck on the `promise` library that was a way of making it more JS like (as it was a port from there).. I believe the new graphql-core uses asyncio more..

(I could be wrong.. but this thing floats around my head a bit too)

**Comment by oliver-sanders** [comment]
@dwsutherland, yep.

I've put some notes here: https://github.com/cylc/cylc-uiserver/issues/333

tldr;

* GraphQL core v3 is a complete re-write of GraphQL core v2, so about as much effort as porting to another server.
* Other servers exist, but it would appear they have GraphQL core as a dependency so are likely wrappers that provide integration support.
* There are no active servers that support Tornado so we will need to develop a bespoke integration for this bit at least.
* It's possible to stick with Graphene, but we could move to another code-driven schema definition framework if it helped.

**Comment by MetRonnie** [comment]
(reminder: do check GH Actions failures before merging)

image
